ui/lib/core/addon/components/shamir/dr-token-flow.hbs

```diff
@@ -0,0 +1,136 @@
+          <h4 class="field-title">
+            One time password (OTP)
+          </h4>
+          <p class="help has-text-grey has-bottom-margin-xs">
+            This OTP will be used to decode the generated operation token. Please save it.
+          </p>
+          <div class="message is-list has-copy-button" tabindex="-1">
+            <HoverCopyButton @copyValue={{this.otp}} />
+            <code class="is-word-break">{{this.otp}}</code>
+          </div>
+        </div>
+      {{/if}}
+      <div class="has-bottom-margin-xl">
+        <h4 class="field-title">
+          DR operation token command
+        </h4>
+        <p class="help has-text-grey has-bottom-margin-xs">
+          {{#if this.otp}}
+            This command contains both the encoded token and the OTP. It should be executed on the secondary cluster in order
+            to generate the operation token.
+          {{else}}
+            This command requires the OTP saved earlier. It should be executed on the secondary cluster in order to generate
+            the operation token.
+          {{/if}}
+        </p>
+        <div class="message is-list has-copy-button" tabindex="-1">
+          {{! template-lint-disable quotes }}
+          {{#let
+            (if
+              this.otp
+              (concat 'vault operator generate-root -dr-token -otp="' this.otp '" -decode="' this.encodedToken '"')
+              (concat 'vault operator generate-root -dr-token -otp="<enter your otp here>" -decode="' this.encodedToken '"')
+            )
+            as |cmd|
+          }}
+            <HoverCopyButton @copyValue={{cmd}} />
+            <code class="is-word-break">{{cmd}}</code>
+          {{/let}}
+          {{! template-lint-enable quotes }}
+        </div>
+      </div>
+    </div>
+    <div>
+      <button type="button" class="button is-primary" {{on "click" this.onCancelClose}}>
+        Clear &amp; Close
+      </button>
+    </div>
+  {{else if this.started}}
+    <Shamir::Form
+      @action={{@action}}
+      @progress={{this.progress}}
+      @threshold={{this.threshold}}
+      @errors={{this.errors}}
+      @inputLabel="Root key portion"
+      @buttonText="Generate token"
+      @onSubmit={{this.onSubmitKey}}
+      @otp={{this.otp}}
+      @alwaysShowProgress={{true}}
+      data-test-dr-token-flow-step="shamir"
+    >
+      <p>Generate an operation token by entering a portion of the
+        <strong>primary's root key</strong>. Once all portions are entered, the generated token may be used to manage your
+        secondary Disaster Recovery cluster.
+      </p>
+    </Shamir::Form>
+  {{else if this.generateWithPGP}}
+    <ChoosePgpKeyForm
+      @onCancel={{fn (mut this.generateWithPGP) false}}
+      @onSubmit={{this.usePgpKey}}
+      @formText={{this.pgpText.form}}
+      @confirmText={{this.pgpText.confirm}}
+      @buttonText="Generate operation token"
+      data-test-dr-token-flow-step="choose-pgp"
+    />
+  {{else}}
+    {{! Not started }}
```

Comment:
Is this a comment? I first read it as Not started this section, almost like a todo?

Reply:
Oh yeah, this is the comment marking that this section should show when the generate token flow is not started

Reply:
Updated so it's more clear